main.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+/* Вимоги до Домашки:
+
+1) Input-поле із кнопкою «Start timeout» — у інпут-полі користувач може ввести кількість відповідних секунд, які залишилися у росії та росіяни до їх зникнення */
+
+const secondsLeft = document.querySelector('.put-seconds-to-destruction');
+const starter = document.querySelector('.btn-timeout-message');
+
+const kremlinOnFire = () => {
+    window.open("https://mayday.rocks/wp-content/uploads/2019/04/maxresdefault-211.jpg");
+}
+
+starter.addEventListener('click', (event) => {
+    event.preventDefault();
+    if(secondsLeft.value === ''){
```

Comment:
```suggestion
    if(secondsLeft.value === '') {
```

Also, it can be simplified 
```suggestion
    if(!secondsLeft.value) {
```